docs/application/vstools/tools/project-wizard.md

```diff
@@ -1,53 +1,53 @@
 ﻿# Project Wizard
 
-The Project Wizard in Visual Studio Tools for Tizen is used to create a Tizen .NET application project.
+You can use Project Wizard to create Tizen .NET application projects. In Project Wizard, you can select a template and a profile to create the project. Aditionally, you can set the basic project properties, such as the project name, package name, location, and so on.
 
-In the Project Wizard, you can select a template and profile to create the project. In addition, you can set basic project properties, such as the project name, package name, and location.
+The Project Wizard provides the following project templates using which you can start a Tizen application project:
```

Comment:
Sure, I will modify..... to avoid repeated "Tizen" word.

In my opinion all the three sentences convey the same meaning. Therefore, I would retain the following:

The Project Wizard provides the following project templates using which you can start a Tizen .NET application project:

Reply:
I'm not sure, but is the grammar correct?

The skeleton of this sentence is "The Wizard provides the templates using which you can start a project:"

I think the original sentence's meaning was "The wizard provides the templates and you can use it to start a project", but with the updated, I'm not sure. So, would you guide me the grammar of this sentence?